app/controllers/v0/facilities/cemetery_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+class V0::Facilities::CemeteryController < FacilitiesController
+  before_action :validate_params, only: [:index]
+
+  # Index supports the following query parameters:
+  # @param bbox - Bounding box in form "xmin,ymin,xmax,ymax" in Lat/Long coordinates
+  # @param type - Optional facility type, values = all (default), health, benefits, cemetery
+  def index
+    results = VACemeteryFacility.query(bbox: params[:bbox])
+    render json: results,
+           serializer: CollectionSerializer,
+           each_serializer: VACemeteryFacilitySerializer
+  end
+
+  def show
+    results = VACemeteryFacility.find_by_id(id: params[:id])
```

Comment:
@mphprogrammer I think is a #find_by_id that they created themselves on VACemeteryFacility object, not part of ActiveRecord, but it is a little confusing.

FWIW, ActiveRecord moved away from this nomenclature in their FinderMethod mixin, and started using `find_by(params = {})`  so you would need to do `find_by(id: id)` if following that paradigm.